isotovideo

```diff
@@ -22,6 +22,11 @@ Enable direct output to STDERR instead of autoinst-log.txt
 
 isotovideo will chdir to that directory on startup
 
+=item B<--color=[yes|no|auto]>
+
+Enable or disable color output explicitly. Defaults to "auto". Alternatively
```

Comment:
As you implemented it, the default is `yes`.
`auto` usually means that colors should be printed only if the output goes to a terminal (not piped or redirected to a file).

So if we document that and call it `auto`, then it IMHO should behave like that.
Compare `man git-config` and look for `color.ui`

Reply:
btw, during testing it seems both STDOUT and STDERR are seen as not printing to a terminal, so we can't really test that.

But regarding implementation, something like:
```
# if one of the two filehandles is not opened to a tty, don't use colors
if ($options{color} eq 'auto' and (not -t STDOUT or not -t STDERR)) {
    $ENV{ANSI_COLORS_DISABLED} = 1;
}
```

Reply:
I extended the condition now with what you suggested. The "default" test is a TODO... since I don't know how we can test it, but this way at least the code paths gets exercised...

Reply:
Ok. With the changed default (currently we always get colored output), it would be good to run an openQA test to ensure we still get a colored autoinst-log.txt